inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -677,6 +697,9 @@ void ngfunction_2_irv10(pugi::xml_node& netXml,
         pugi::xml_node data = layer.append_child("data");
         std::string node_type_name{node->get_type_name()};
 
+        // must be executed before visit_attributes which can change values
+        layer_type_attribute.set_value(translate_type_name(node_type_name).c_str());
```

Comment:
Why can  some visit_attributes  change the values in the serialization pass?

Reply:
Visitor for FrameworkNode changes opset name parameter.